contrib/xfce4-settings/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "xfce4-settings"
+pkgver = "4.18.4"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-pluggable-dialogs",
+    "--enable-sound-settings",
+    "--enable-upower-glib",
+]
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gettext-devel",
+    "gmake",
+    "intltool",
+    "libtool",
+    "pkgconf",
+    "xfce4-dev-tools",
+]
+makedepends = [
+    "colord-devel",
+    "exo-devel",
+    "fontconfig-devel",
+    "garcon-devel",
+    "glib-devel",
+    "gtk+3-devel",
+    "libnotify-devel",
+    "libx11-devel",
+    "libxcursor-devel",
+    "libxfce4ui-devel",
+    "libxfce4util-devel",
+    "libxi-devel",
+    "libxklavier-devel",
+    "libxrandr-devel",
+    "upower-devel",
+    "xfconf-devel",
+    "xorgproto",
+    "xserver-xorg-input-libinput-devel",
+]
+# See patches/defaults.patch
+depends = ["adw-gtk3", "papirus-icon-theme"]
```

Comment:
i'd make these softdeps via a subpackage

Reply:
what would i call the subpackage?

Reply:
i went with `-themes` for both `xfwm4` and `xfce4-settings`, though it sounds a bit awkward